gobblin-runtime/src/test/java/org/apache/gobblin/runtime/job_monitor/MockedKafkaJobMonitor.java

```diff
@@ -99,24 +103,32 @@ public Void answer(InvocationOnMock invocation)
   }
 
   @Override
-  public Collection<Either<JobSpec, URI>> parseJobSpec(byte[] message)
+  public Collection<JobSpec> parseJobSpec(byte[] message)
```

Comment:
Should avoid changing the public API that has existed for almost 2 years.

Reply:
changed back.